src/content/post/class7.md

```diff
@@ -8,6 +8,75 @@ slug = "class7"
 
 ## Motivation
 ...
+##
+## Investigating Ad Transparency Mechanisms in Social Media: A Case Study of Facebook’s Explanations
+> Andreou, Athanasios, et al. "Investigating ad transparency mechanisms in social media: A case study of Facebook’s explanations." NDSS, 2018.[[PDF]](http://wp.internetsociety.org/ndss/wp-content/uploads/sites/25/2018/02/ndss2018_10-1_Andreou_paper.pdf)
+
+### What is ad transparency mechanisms in social media？ 
+
+Transparency mechanisms are solutions for many privacy complaints from users and policy makers. Users have little understanding of what data the advertising platforms have about them and why they are shown particular ads. The transparency mechanisms in Facebook are "Why am I seeing this?" botton that provides  users with an explanation of why they were shown a particular ad (ad explanations), and an Ad Preferences Page that provides users with a list of attributes Facebook has inferred about them and how (data explanations). In Twitter, there are similar transparency mechanisms. You can also see "Why am I seeing this?" botton that provides  users with an explanation of why they were shown a particular ad (ad explanations) in Twitter.
+
+### What did this paper do?
+
+This paper mainly did an investigation and analysis of ad explanations, or why users see a certain ad and an investigation of data explanations, or how data is inferred about a user, which is strongly related to the ad transparency mechanisms in Facebook. This paper first introduced how ad platform in facebook works and then evaluate the two transparency mechanisms we introduced before with some properties.
+
+### The Ad platform in Facebook
+
+There are three main processes in facebook ad platform: a) the data inference process; b) the audience selection process; c) the user-ad matching process.
+
+(a) The data inference process is the process that allows the advertising platform to learn the users’ attributes. It has three parts: (1) the raw user data (the inputs), containing the information the
+advertising platform collects about a user either online or offline; (2) the data inference algorithm (the mapping function between inputs and outputs), covering the algorithm the advertising platform uses to translate input user data to targeting attributes;(3) the resulting targeting attributes (the outputs) of each user that advertisers can specify to select different groups of users.
+
+![](https://github.com/jindingars/secML.github.io/blob/master/src/content/images/a.png)
```

Comment:
Please change the link to `/images/class7/a.png` and add the image to the `src/content/images/class7` folder.  The images may not render properly otherwise.